src/components/vocabularies/DeleteVocabularyForm.tsx

```diff
@@ -44,7 +44,7 @@ const DeleteVocabularyForm: React.FC<DeleteVocabularyFormProps> = ({
       onSubmit={onSubmit}
     >
       <Hidden name="workspaceId" value={workspace?.id || ""} />
-      <Hidden name="vocabularyId" value={vocabulary?.id || ""} />
+      <Hidden name="vocabularyIri" value={vocabulary?.uri || ""} />
```

Comment:
here

Reply:
That's not so simple. The discrepancy is a result of historical API of SGoV and chances are that if I were to replace `uri` with `iri` some endpoint will start failing...

Reply:
We should be using IRI everywhere ideally.